### PR TITLE
fix bash exit on empty argument

### PR DESCRIPTION
**Pull request by toschneck**
Currently if a users missed to specify a typ on e.g. `kdes` the bash exit due the sourced `exit 1` command.
Fixed this with `return 1` and nested if. Maybe not the elegant way, but seems to be a safe way ;-)

### CODE REVIEW

**Review by scheeles — APPROVED** [review]
LGTM

File: fubectl.source [hunk 1]
```diff
@@ -2,13 +2,15 @@
 alias _inline_fzf="fzf --multi --ansi -i -1 --height=50% --reverse -0 --header-lines=1 --inline-info --border"
 alias _inline_fzf_nh="fzf --multi --ansi -i -1 --height=50% --reverse -0 --inline-info --border"
 
-_isClusterSpaceObject() {
+_isObjectSpecified(){
```
Comment (thz) [inline fubectl.source:5]: Quite some added "complexity" for a simple `[ -z "$foo" ] && echo "please specify foo"`
Also the name does not indicate that the main purpose of the function is the log message.

File: fubectl.source [hunk 2]
```diff
@@ -34,10 +36,12 @@ alias kp="xdg-open 'http://localhost:8001/api/v1/namespaces/kube-system/services
 
 # [kwatch] watch resource
 kwatch() {
-    if _isClusterSpaceObject $1 ; then
-        kubectl get "${1}" | _inline_fzf | awk '{print $1}' | xargs watch kubectl get "${1}"
-    else
-        kubectl get "${1}" --all-namespaces | _inline_fzf | awk '{print $1, $2}' | xargs watch kubectl get "${1}" -n
+    if _isObjectSpecified $1 ; then
```
Comment (thz) [inline fubectl.source:39]: Can we avoid the big nesting braces?
I'd prefer something like `[ -z "$1" ] && fail "xyz not specified."` this allows a more specific error message and uses a lot less code.

**Review by thz — CHANGES_REQUESTED** [review]

**Comment by thz** [comment]
I'll send the respective counter proposal.

**Comment by thz** [comment]
Obsoleted by #26

**Pull request closed without merging**